src/PKSim.UI.Starter/ApplicationStartup.cs

```diff
@@ -54,24 +75,93 @@ private static IContainer initializeForStartup(IShell shell)
+            container.Register<IOriginDataParameterToParameterDTOMapper, OriginDataParameterToParameterDTOMapper>();
+            container.Register<IIndividualSettingsDTOToIndividualMapper, IndividualSettingsDTOToIndividualMapper>();
+            container.Register<IEditValueOriginPresenter, EditValueOriginPresenter>();
+            container.Register<IEditValueOriginView, EditValueOriginView>();
+            container.Register<ValueOriginBinder<ValueOriginDTO>, ValueOriginBinder<ValueOriginDTO>>();
+            container.Register<IValueOriginPresenter, ValueOriginPresenter>();
+            container.Register<IValueOriginView, ValueOriginView>();
+            container.Register<OSPSuite.UI.Services.IToolTipCreator, ToolTipCreator>();
+            container.Register<IIndividualParametersPresenter, IndividualParametersPresenter>();
+            container.Register<IIndividualParametersView, IndividualParametersView>();
+            container.Register<IParameterGroupsPresenter, ParameterGroupsPresenter>();
+            container.Register<IParameterGroupsView, ParameterGroupsView>();
+            container.Register<IParameterGroupNodeCreator, ParameterGroupNodeCreator>();
+            container.Register<IParameterGroupToTreeNodeMapper, ParameterGroupToTreeNodeMapper>();
+            container.Register<ITreeNodeFactory, TreeNodeFactory>();
+            container.Register<IDynamicGroupExpander, DynamicGroupExpander>();
+            container.Register<IParameterContainerToTreeNodeMapper, ParameterContainerToTreeNodeMapper>();
+            container.Register<INodeToCustomizableParametersPresenterMapper, NodeToCustomizableParametersPresenterMapper>();
+            container.Register<IContainerToCustomableParametersPresenterMapper, ContainerToCustomableParametersPresenterMapper>();
+            container.Register<IParameterGroupToCustomizableParametersPresenter, ParameterGroupToCustomizableParametersPresenter>();
+            container.RegisterFactory<IMultiParameterEditPresenterFactory>();
+            container.Register<INoItemInSelectionPresenter, NoItemInSelectionPresenter>();
+            container.Register<INoItemInSelectionView, NoItemInSelectionView>();
+            container.Register<IUserSettings, UserSettings>();
+            container.Register<INumericFormatterOptions, NumericFormatterOptions>();
+            container.Register<ISkinManager, SkinManager>();
+            container.Register<DirectoryMapSettings, DirectoryMapSettings>();
+            container.Register<IPresentationSettingsTask, PresentationSettingsTask>();
+            container.Register<IWorkspace, Infrastructure.Workspace>();
+            container.Register<IWithWorkspaceLayout, Infrastructure.Workspace>();
+            container.Register<IWorkspaceLayout, WorkspaceLayout>();
+            container.Register<ITreeNodeContextMenuFactory, TreeNodeContextMenuFactory>();
+            container.Register<IIndividualMoleculesPresenter, IndividualMoleculesPresenter>();
```

Comment:
It might be possible to create a static, simple presenter with no dependencies, like a NullIndividualMoleculesPresenter or something. Then when it's used it could be checked against that static instance to see if the tab should be displayed.

Reply:
OK I am looking into it